src/Persistence.php

```diff
@@ -27,14 +27,10 @@ class Persistence
     /**
      * Connects database.
      *
-     * @param string $dsn      Format as PDO DSN or use "mysql://user:pass@host/db;option=blah", leaving user and password arguments = null
-     * @param string $user
-     * @param string $password
-     * @param array  $args
-     *
-     * @return Persistence
+     * @param string $dsn Format as PDO DSN or use "mysql://user:pass@host/db;option=blah",
+     *                    leaving user and password arguments = null
      */
-    public static function connect($dsn, $user = null, $password = null, $args = [])
+    public static function connect(string $dsn, string $user = null, string $password = null, array $args = []): self
```

Comment:
dns may be an array according to https://github.com/atk4/dsql/blob/develop/src/Connection.php#L77